packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/data_triggers/bindings/list/v0/mod.rs

```diff
@@ -0,0 +1,131 @@
+use crate::execution::validation::state_transition::documents_batch::data_triggers::triggers::dashpay::create_contact_request_data_trigger;
+use crate::execution::validation::state_transition::documents_batch::data_triggers::triggers::dpns::create_domain_data_trigger;
+use crate::execution::validation::state_transition::documents_batch::data_triggers::triggers::feature_flags::create_feature_flag_data_trigger;
+use crate::execution::validation::state_transition::documents_batch::data_triggers::triggers::reject::reject_data_trigger;
+use crate::execution::validation::state_transition::documents_batch::data_triggers::triggers::reward_share::create_masternode_reward_shares_data_trigger;
+use crate::execution::validation::state_transition::documents_batch::data_triggers::triggers::withdrawals::delete_withdrawal_data_trigger;
+
+use dpp::state_transition_action::document::documents_batch::document_transition::{DocumentTransitionAction, DocumentTransitionActionType};
+use dpp::system_data_contracts::feature_flags_contract::document_types::update_consensus_params;
+use dpp::system_data_contracts::withdrawals_contract::document_types::withdrawal;
+use dpp::system_data_contracts::{
+    dashpay_contract, dpns_contract,
+    SystemDataContract,
+};
+use dpp::{errors::ProtocolError};
+use crate::execution::validation::state_transition::documents_batch::data_triggers::bindings::data_trigger_binding::DataTriggerBindingV0;
+
+/// Retrieves a list of data triggers binding with matching params.
+///
+/// This function gets all known data triggers which are then returned
+/// as a vector of `DataTrigger` structs.
+///
+/// # Returns
+///
+/// A `Vec<DataTriggerBinding>` containing all known data triggers.
+///
+/// # Errors
+///
+/// Returns a `ProtocolError` if there was an error.
+pub fn data_trigger_bindings_list_v0() -> Result<Vec<DataTriggerBindingV0>, ProtocolError> {
```

Comment:
I'm not sure this is the correct way... The list should be the list of bindings, not the list of the versions of bindings. So I believe each binding in this list should be called without the version determined by "data_trigger_bindings_list_v0", hence we would go to "data_trigger_bindings_list_v1" when we add a new data trigger, not we update the version of a data trigger.

Reply:
Triggers are versioned here internally, but if we need to change binding we highly likely need to change the definitions as well.

Reply:
Understood